### DISCUSSION TITLE
Connect arrow "spawns" right behind another node, making it impossible to find/adjust without moving everything

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1488] [Help] [answered]
<img width="457" alt="dsfksdfisdjfdsa" src="https://github.com/freeplane/freeplane/assets/147225942/39d5ac1f-c802-4f96-81d2-6e05132c93bf">
<img width="477" alt="sadfjsdjiifdsafd" src="https://github.com/freeplane/freeplane/assets/147225942/4f9cd499-5385-42d0-81f8-9861f18b20ab">

As you can see, the arrow is completely hidden

**Comment by macmarrum** [comment, marked as answer]
`Preferences…->Defaults->Connectors->Paint connectors behind nodes`

**Reply by adamfreeplaneuser01** [reply]
Easy fix, thanks!
